styles.css

```diff
@@ -1 +1,212 @@
-body
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Merriweather&family=Roboto&display=swap');
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+body {
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 24px;
+    color: #0C0C0C;
+}
+
+h1 {
+    font-family: "Merriweather", "Times New Roman", serif;
+    color: white;
+    font-size: 56px;
+}
+
+h2 {
+    font-family: "Merriweather", "Times New Roman", serif;
+    color: #5722CD;
+}
+
+h3 {
+    font-family: "Merriweather", "Times New Roman", serif;
+}
+
+h4 {
+    color: white;
+    font-size: 28px;
+    font-weight: lighter;
+}
+
+h5 {
+    color: #5722CD;
+    font-family: "Merriweather", "Times New Roman", serif;
+    font-size: 28px;
+}
+
+nav a:hover {
+    color: #7A7A7A;
+}
+
+nav a:visited {
+    color: white;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-content: space-between;
+    list-style-type: none;
+    font-size: 28px;
+    gap: 50px;
+    margin-right: 100px;
```

Comment:
Creatief, maar da's valsspelen 🤓 Check vooral de opname waarin we de opdracht bespreken om te kijken hoe je dit het best op kunt lossen!